core/storage/feedback/gae_models.py

```diff
@@ -231,14 +329,22 @@ class FeedbackAnalyticsModel(base_models.BaseMapReduceBatchResultsModel):
 
     The key of each instance is the exploration id.
     """
-    # The number of open feedback threads filed against this exploration.
+    # The number of open feedback threads for this exploration.
     num_open_threads = ndb.IntegerProperty(default=None, indexed=True)
-    # Total number of feedback threads filed against this exploration.
+    # Total number of feedback threads for this exploration.
     num_total_threads = ndb.IntegerProperty(default=None, indexed=True)
 
     @classmethod
     def create(cls, model_id, num_open_threads, num_total_threads):
-        """Creates a new FeedbackAnalyticsModel entry."""
+        """Creates a new FeedbackAnalyticsModel entry.
+
+        Args:
+            num_open_threads: int. number of open feedback threads for
+                this exploration.
+            num_total_threads: int. total number of feedback threads for
+                this exploration.
+            model_id: str. ID of the model instance.
```

Comment:
Why are the args in the wrong order?

ID of the model instance to be created. This is the same as the exploration ID.